### ISSUE TITLE
Wrong image/ horizontal break sizes

### ISSUE THREAD

**Issue by cyanit** [elementor/elementor#9463]
## Prerequisites

- [ x ] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [ x ] The issue still exists against the latest stable version of Elementor.

## Description

Okay, so I notices something strange: first my layout went nuclear out of nothing, borders looked fine. Then I coulnt properly resize inner sections. It could be similar/related to issue #4679.

Lets start with what the builder is showing:
<img width="869" alt="preview" src="https://user-images.githubusercontent.com/26248671/68070272-a818be80-fd6c-11e9-9efe-681558714609.png">

The images are resized correctly to the boxes.

Now heres the preview (and what it lookes like on the final site:
<img width="866" alt="result-2" src="https://user-images.githubusercontent.com/26248671/68070289-ef06b400-fd6c-11e9-8be3-6c76976dff59.png">

The images are either in the size set in the Elementor content editor (in case of mine) or in their original size (in case of tumpperinis).

A look into the chrome inspector confirmed this:
![Chrome Selector](https://user-images.githubusercontent.com/26248671/68070310-24ab9d00-fd6d-11e9-9799-acf0984e622e.png)

![Elementor content editor](https://user-images.githubusercontent.com/26248671/68070316-342ae600-fd6d-11e9-91d6-983377401864.png)

My first idea was that this is the column size of the inner section:
![image](https://user-images.githubusercontent.com/26248671/68070373-97b51380-fd6d-11e9-9581-dd53acb39a7e.png)

But i noticed something strange: it was not an integer in the textfield. So I tried to set it to the correct one, which did not work. One below neither.

<img width="62" alt="i entered 78" src="https://user-images.githubusercontent.com/26248671/68070387-c501c180-fd6d-11e9-9c3d-1dec528a7796.png">

## TLDR
- images are at the wrong size (resizing isn't taking place?)
- seems like this is due to a problem with general resizing and/or resizing due to inner section restrictions

## Steps to reproduce

no idea how to do that right now. This just struck me.

## Isolating the problem

- [x] This bug happens with only Elementor plugin active (and Elementor Pro).
- [x] This bug happens with a default WordPress theme active.
- [ ] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>

![image](https://user-images.githubusercontent.com/26248671/68070620-21b2ab80-fd71-11e9-8178-5708fb8760bc.png)

![image](https://user-images.githubusercontent.com/26248671/68070631-3131f480-fd71-11e9-9d70-7d8dd8b89239.png)

</details>

**Comment by bainternet** [comment]
@cyanit 

we can't replicate.

Please contact our support at support @elementor.com for guidance in resolving this issue and mention that you were directed by me and reference this issue.

thanks.
